services/movie.py

```diff
@@ -1,11 +1,13 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Consider using `Optional[list[int]]` for type hinting to indicate that `genres_ids` and `actors_ids` can be `None`.